File: leetcode_solution/convert-the-temperature.py
class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
        kelvin=celsius+273.15
        Fahrenheit = celsius * 1.80 + 32.00
        l=[]
        l.append(kelvin)
        l.append(Fahrenheit)
        return l
        

File: ConvertTheTemperature.cpp
class Solution {
public:
    vector<double> convertTemperature(double celsius) {
        vector<double>result;
         double kelvin=celsius+273.15;
         double fahr=celsius*1.80+32.00;
         result.push_back(kelvin);
          result.push_back(fahr);
         return result;
    }
};